api/RTCPeerConnection.json

```diff
@@ -395,10 +395,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnection/addTransceiver",
           "support": {
             "chrome": {
-              "version_added": "69"
+              "version_added": "72"
```

Comment:
Actually, this did land in 69.
https://chromium.googlesource.com/chromium/src/+/16b2a7fb36f48ea5f69d856aea2b244936140202%5E%21/#F70
https://storage.googleapis.com/chromium-find-releases-static/16b.html#16b2a7fb36f48ea5f69d856aea2b244936140202

Reply:
Hm, I could have sworn it was disabled or something. OK, will fix.